waspc/e2e-test/Tests/WaspComplexTest.hs

```diff
@@ -228,3 +229,28 @@ insertCodeIntoWaspFileAfterVersion = insertCodeIntoWaspFileAtLineNumber lineNumb
   where
     lineNumberInWaspFileAfterVersion :: Int
     lineNumberInWaspFileAfterVersion = 5
+
+addApi :: ShellCommandBuilder [ShellCommand]
+addApi = do
+  sequence
+    [ appendToWaspFile apiDecl,
+      createFile apiFile "./src/server" "apis.ts"
+    ]
+  where
+    apiDecl =
+      unlines
+        [ "api fooBar {",
+          "  fn: import { fooBar } from \"@server/apis.js\",",
```

Comment:
I would maybe suggest adding another authenticated API route (it can have the same JS impl)